Formula/pioneer.rb

```diff
@@ -3,6 +3,7 @@ class Pioneer < Formula
   homepage "https://pioneerspacesim.net/"
   url "https://github.com/pioneerspacesim/pioneer/archive/20200203.tar.gz"
   sha256 "3055d63c1bd3377c3794eee830a8adbd650b178bad9e927531e38cb5d5838694"
+  license "GPL-3.0"
```

Comment:
How did you determine `GPL-3.0`? I see a whole list of licenses in the [`licenses` directory](https://github.com/pioneerspacesim/pioneer/tree/master/licenses) but I can't find if there are any indicators of which ones apply where

Reply:
They put in the `authors.txt`, https://github.com/pioneerspacesim/pioneer/blob/master/AUTHORS.txt#L127-L132

Also, this is a good case, the different parts of the codebase got licensed under different licenses.

Reply:
Ah yes. Missed the `AUTHORS.txt` file. Thanks!